### PR TITLE
Added format option and recursive mode for directories scanning

### PR DESCRIPTION
**Pull request by mscasso**
Also, help was improved to show the new command options. Makefile was changed to be consistent.

### CODE REVIEW

File: src/scanner.c [hunk 1]
```diff
@@ -44,6 +45,9 @@
 
 char *wfp_buffer;
 
+bool recursive = false;
+char format[9] = "plain";
```
Comment (scanossdev) [inline src/scanner.c:49]: it would take 10 bytes to store cyclonedx + chr(0)

**Review by scanossdev — COMMENTED** [review]
check scanner.c comment

File: src/scanner.c (hunk 1 above)
Comment (mscasso) [reply src/scanner.c]: Ups, you are right. Corrected at last commit.

**Comment by scanossdev** [comment]
Awesome, thanks!

**Pull request merged**